kii_thing_if.c

```diff
@@ -1343,22 +1368,12 @@ kii_bool_t get_thing_type(
             char resource_path[128];
             kii_t* kii = &(kii_thing_if->command_handler);
 
-            if (sizeof(resource_path) / sizeof(resource_path[0]) <=
-                    CONST_STRLEN(THING_IF_APP_PATH) +
-                    strlen(kii->kii_core.app_id) + CONST_STRLEN(THINGS_PART) +
-                    strlen(kii->kii_core.author.author_id) +
-                    CONST_STRLEN(THING_TYPE_PART)) {
-                M_KII_LOG(kii->kii_core.logger_cb(
-                        "resource path is longer than expected.\n"));
-                M_KII_THING_IF_ASSERT(0);
+            if (prv_set_thing_type_resource_path(
```

Comment:
@satoshikumano That's true. I replace `kii` to `kii->kii_core.app_id` and `kii->kii_core.author.author_id`.